src/main/java/com/bwfcwalshy/flarebot/github/GithubListener.java

```diff
@@ -15,16 +15,43 @@ public void handle(PushEvent e) {
             return;
         EmbedBuilder eb = MessageUtils.getEmbed();
         eb.setAuthor(e.getSender().getLogin(), e.getSender().getProfile(), e.getSender().getAvatarUrl());
-        StringBuilder sb = new StringBuilder();
         for (Commit commit : e.getCommits()) {
-            sb.append(String.format("`%s` in `%s` - [`%s`](%s) %s\n",
-                    commit.getAuthor().getName(),
-                    e.getRef().substring(e.getRef().lastIndexOf('/') + 1),
-                    commit.getId().substring(0, 7),
-                    commit.getUrl(),
-                    commit.getMessage()));
+            eb.addField("Commit:", "[" +
+                    commit.getId().substring(0, 7) + "](" +
+                    commit.getUrl() + ")\n Branch `" +
+                    e.getRef().substring(e.getRef().lastIndexOf('/') + 1) + "` " + "```" +
+                    commit.getMessage() + "```", false);
+
+            if (commit.getAdded().length > 0) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("```Markdown\n");
+                for (String file : commit.getAdded()) {
+                    sb.append("* " + file + "\n\n");
+                }
+                String added = sb.toString() + "```";
+                eb.addField("Added files", added, false);
+            }
+
+            if (commit.getRemoved().length > 0) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("```Markdown\n");
+                for (String file : commit.getRemoved()) {
+                    sb.append("* " + file + "\n\n");
+                }
+                String removed = sb.toString() + "```";
+                eb.addField("Removed files", removed, false);
+            }
+
+            if (commit.getModified().length > 0) {
+                StringBuilder sb = new StringBuilder();
+                sb.append("```Markdown\n");
+                for (String file : commit.getModified()) {
+                    sb.append("* " + file + "\n\n");
+                }
+                String modified = sb.toString() + "```";
+                eb.addField("Modified files", modified, false);
```

Comment:
I reject that, it will just look horrible. But with taking up less space I'd say limit the amount of files shown for each one to 5 max and put a ... after it it's more.